plugins/plugin-docker/che-plugin-docker-machine/src/main/java/org/eclipse/che/plugin/docker/machine/ComposeMachineProviderImpl.java

```diff
@@ -590,10 +595,22 @@ private void readContainerLogsInSeparateThread(String container,
                 } catch (ContainerNotFoundException e) {
                     isContainerRunning = false;
                 } catch (IOException e) {
-                    LOG.error("Failed to get logs from machine {} of workspace {} backed by container {}",
+                    LOG.error("Failed to get logs from machine {} of workspace {} backed by container {}, because: {}. Cause: {}",
```

Comment:
Maybe it would be better to print error in the log after 5 five failed attempts...